plugins/woocommerce-admin/client/subscriptions/index.tsx

```diff
@@ -72,11 +104,12 @@ const GetStartedButton = () => {
 							 * Navigate to either newSubscriptionProductUrl or onboardingUrl
 							 * depending on the which treatment the user is assigned to.
 							 */
+							// eslint-disable-next-line no-console
```

Comment:
Adding this was necessary to get over WC's linting on commit. It can be removed once we remove the following console log.